tests/test_data/carry4/carry4.pcf

```diff
@@ -0,0 +1,48 @@
+set_io a[0]  U14
```

Comment:
The PCF might not be needed as there already is the XDC with the constraints.

Reply:
I ran the tests locally and got `FileNotFoundError: [Errno 2] No such file or directory: '/home/jaromharris/symbiflow-xc-fasm2bels/tests/test_data/carry4/carry4.pcf'`, so I think it is required.

Reply:
Right, I forgot that the test is actually using both formats.